authentikos/test/integ-simple.sh

```diff
@@ -68,6 +68,7 @@ main() {
   kubectl wait deployment authentikos --for="condition=available" --timeout="$timout"
 
   run_test "$(with_timeout get_tokeninfo "$timout")"
+  kubectl logs -l app=authentikos
```

Comment:
Currently if `run_test` fails one of its checks the script exits immediately. I think we can do something like the following to instead capture the error code to return after the log dump.
```suggestion
  local code=0
  run_test "$(with_timeout get_tokeninfo "$timout")" || code="$?"
  kubectl logs -l app=authentikos
  exit "${code}"
```

Reply:
I am good Cole! I hope you are doing well also! I like the suggestion. Let me make the update.

Reply:
Updated with minor tweaks to the syntax to get the desired behavior. PTAL